Kickstarter-iOS/Features/RewardsCollection/Controller/RewardsCollectionViewControllerTests.swift

```diff
@@ -44,6 +44,7 @@ final class RewardsCollectionViewControllerTests: TestCase {
   func testRewards_NonBacker_LiveProject_Landscape() {
     let project = Project.cosmicSurgery
       |> Project.lens.state .~ .live
+      |> Project.lens.rewardData.rewards %~ { Array($0[1...3]) }
```

Comment:
This just reduces the number of items so that the unavailable one shows as well, even when it's last